### DISCUSSION TITLE
[question]请问设计器的相关资料哪里可以找到呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#873] [Q&A] [answered]
请问设计器的相关资料哪里可以找到呢？
在官方网站这里看到2个相关的软连接，但是都没有具体的指向。
![20241023103223](https://github.com/user-attachments/assets/74b86ab4-9ada-4bde-ab85-d54264ccb27d)

**Comment by chilingling** [comment, marked as answer]
学院课程：https://opentiny.design/tiny-engine#/help-center/course/engine
开发协议规范：https://opentiny.design/tiny-engine#/protocol
